metrics/distances.py

```diff
@@ -0,0 +1,86 @@
+from votekit.profile import PreferenceProfile
+import numpy as np
+import ot
```

Comment:
make sure whatever library this is gets added to pyproject.toml

Reply:
done @jamesturk , do I commit the pyproject.toml ?